spec/operations/story_operations/pusher_notification_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe StoryOperations::PusherNotification do
+  class PusherClass
+    include StoryOperations::PusherNotification
+    attr_reader :user, :model
+
+    def initialize(model, user)
+      @model = model
+      @user = user
+    end
+  end
+
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
```

Comment:
you're not using this, if the record i required add in a before block, if is not just remove it